hms-app/src/components/input-forms/IdeaForm.tsx

```diff
@@ -202,6 +206,11 @@ function IdeaForm(props: IProps) {
               <Textarea
                 label="Title"
                 required
+                error={
+                  ideaText.title.length > allowedIdeaTitleLength
+                    ? 'max 100 words'
```

Comment:
the max is 100 chars not words :P and instead of "100" use a reference to the max length value

Reply:
updated